nestkernel/event.h

```diff
@@ -827,13 +843,20 @@ class GapJEvent : public SecondaryEvent
   static void
   set_syn_id( const synindex synid )
```

Comment:
You are right. But this also means that `set...` is only called once, when the connection type is registered, isn't it? So I would add an `assert( supported_syn_ids_.empty() );` to the `set_...` function, plus add doxygen comments explaining the roles of set and add, respectively.